deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -193,6 +193,11 @@ def GenerateConfig(context):
 echo "Starting services."
 systemctl start cloudsqlproxy
 sleep 5
+
+echo "Attempting to upgrade database."
+python $USER_HOME/forseti-security/install/gcp/upgrade_tools/db_migrator.py
+sleep 5
```

Comment:
Why `sleep 5`.... can it take more than 5 seconds?

Reply:
The migrator script should block until the upgrade is finished, removed the sleep statement.